### PR TITLE
fix: fix site configuration issue

### PR DESCRIPTION
**Pull request by Faraz32123**
- site configuration was not being created during tutor dev launch, fixed it.
- add support for using custom credentials repository using CREDENTIALS_REPOSITORY and CREDENTIALS_REPOSITORY_VERSION.
- add command in readme for creating credentials superuser.

### CODE REVIEW

File: tutorcredentials/__about__.py
Comment (regisb) [inline tutorcredentials/__about__.py:1]: Please don't bump the version number here! I'll recap the process:

- Open a first PR with just your changes:
    - Include just your operating changes
    - Create a changelog entry with `scriv create`
    - Discuss until your PR is merged
- If you want to publish a new version, then open a new PR:
    - bump the version number in `__about__.py`
    - collect changelog entries with `scriv collect`
    - `git commit -a -m v16.xyz`
    - Open a new PR. You can merge that PR without approval, since it does not contain any major change.

If you bump the version number in the first PR, then other committers will have git conflicts. It will also be more difficult to cherry-pick changes in other release branches.

File: tutorcredentials/plugin.py
Comment (regisb) [inline tutorcredentials/plugin.py:25]: This will work only locally, but not in production. On the other hand, it does not make sense to allow the user to modify this setting, so we should remove it from the config and hardcode the value in the init script. That value should probably be something like:

```
{% if ENABLE_HTTPS %}https{% else %}http{% endif %}{{ DISCOVERY_HOST }}/api/v1/
```

Comment (Faraz32123) [reply tutorcredentials/plugin.py:25]: good one

**Review by regisb — CHANGES_REQUESTED** [review]

File: tutorcredentials/plugin.py
Comment (regisb) [inline tutorcredentials/plugin.py:52]: nit: switch to openedx/credentials

**Pull request merged**
